Gemfile

```diff
@@ -122,7 +122,7 @@ gem 'ruby-msg', '~> 1.5.0', :git => 'https://github.com/mysociety/ruby-msg.git',
 gem 'rubyzip', '~> 1.3.0', '< 2.0.0'
 gem 'secure_headers', '~> 3.9.0'
 gem 'statistics2', '~> 0.54'
-gem 'strip_attributes', :git => 'https://github.com/mysociety/strip_attributes.git', :ref => 'c1c14da'
+gem 'strip_attributes', :git => 'https://github.com/mysociety/strip_attributes.git', :ref => rails_upgrade? ? 'e37fbdc' : 'c1c14da'
```

Comment:
RE: https://github.com/mysociety/strip_attributes/commits/globalize3-1.8, I think we should cherry-pick our custom commits on top of the latest master (as per [`globalize3`](https://github.com/mysociety/strip_attributes/commits/globalize3)) rather than merging master in, so that it's really clear what changes we're adding. Looking at [`globalize3-1.8`](https://github.com/mysociety/strip_attributes/commits/globalize3-1.8), I have to go back a page and notice that there are a couple of commits by us added. With [`globalize3`](https://github.com/mysociety/strip_attributes/commits/globalize3), I can immediately see our commits at the top of the branch.

Reply:
I didn't want to do anything to commit `c1c14da` as that is what has been in the Gemfile for quite awhile now. I suppose we could create a new branch or just have master diverge from upstream. Anyway there are failing [specs](https://travis-ci.org/github/mysociety/strip_attributes/builds/696183197) which needs addressing so will look at that and this will need to change with the new branch/commit ref

Reply:
Having a `globalize3/rails-x.x` branch seems like a good idea as you've done.